packages/kolibri/__tests__/heartbeat.spec.js

```diff
@@ -1,27 +1,25 @@
 import mock from 'xhr-mock';
-import coreStore from 'kolibri/store';
 import redirectBrowser from 'kolibri/utils/redirectBrowser';
 import * as serverClock from 'kolibri/utils/serverClock';
 import { get, set } from '@vueuse/core';
 import useSnackbar, { useSnackbarMock } from 'kolibri/composables/useSnackbar'; // eslint-disable-line
 import { ref } from '@vue/composition-api';
 import { DisconnectionErrorCodes } from 'kolibri/constants';
+import useUser, { useUserMock } from 'kolibri/composables/useUser'; // eslint-disable-line
 import { HeartBeat } from '../heartbeat.js';
 import { trs } from '../internal/disconnection';
-import coreModule from '../../../kolibri/core/assets/src/state/modules/core';
 import { stubWindowLocation } from 'testUtils'; // eslint-disable-line
 
 jest.mock('kolibri/utils/redirectBrowser');
 jest.mock('kolibri/urls');
 jest.mock('lockr');
 jest.mock('kolibri/composables/useSnackbar');
-
-coreStore.registerModule('core', coreModule);
+jest.mock('kolibri/composables/useUser');
 
 describe('HeartBeat', function () {
   stubWindowLocation(beforeAll, afterAll);
   // replace the real XHR object with the mock XHR object before each test
-  beforeEach(() => mock.setup());
+  beforeEach(() => useUser.mockImplementation(() => useUserMock()));
```

Comment:
Is there a particular reason for `mock.setup()` not to be called anymore? I'm wondering if this is related to the failing tests